README.md

```diff
@@ -0,0 +1,223 @@
++   import.meta.glob("./**/*.{js,ts,gjs,gts}", { eager: true });
++   start();
++ </script>
+
+- {{content-for "test-body-footer"}}
+```
+
+### Running code replacements on... `app/config/environment.js`
+
+Since the file `app/config/environment.js` is created out of the app blueprint, it has a placeholder `<%= name %>` for your app name. Replace it with the correct value. The name to use can be read in `ENV.modulePrefix` in your `config/environment.js`.
+
+### Running code replacements on... `ember-cli-build.js`
+
+Instead of building the app directly with Broccoli, we use `@embroider/compat`, a module that essentially serves as a bridge between classic apps and modern Vite apps. The Broccoli app instance is still there, but it's passed in argument to Embroider.
+
+Considering an empty 6.2 Ember app, the codemod does the following:
+
+```diff
+ 'use strict';
+
+  const EmberApp = require('ember-cli/lib/broccoli/ember-app');
++ const { compatBuild } = require('@embroider/compat');
+
+  module.exports = function (defaults) {
++   const { buildOnce } = await import('@embroider/vite');
+    const app = new EmberApp(defaults, {
+      // Add options here
+    });
+
+-   return app.toTree();
++   return compatBuild(app, buildOnce);
+  };
+```
+
+### Running code replacements on... `testem.js`
+
+The change in this file introduces a new way to prevent its execution when tests run directly in the browser.
+
+The codemod looks for the `module.exports` and wraps it in a conditional that checks the `module` existence.
+
+```diff
+  'use strict';
+
++ if (typeof module !== 'undefined') {
+    module.exports = { ... }
++ }
+```
+
+### Running code replacements on... `tests/test-helper.js`
+
+If you go back to the modifications done in `tests/index.html`, you can see the new script imports ` { start } from './test-helper`, registers the test files then call `start`. The `start` function is the one the codemod creates at the present step.
+
+Instead of loading the tests then calling `start` from `ember-qunit` directly, we rather export a `start` function that can be called later.
+
+Considering an empty 6.2 Ember app, the codemod does the following:
+
+```diff
+  import Application from 'my-empty-classic-app/app';
+  import config from 'my-empty-classic-app/config/environment';
+  import * as QUnit from 'qunit';
+  import { setApplication } from '@ember/test-helpers';
+  import { setup } from 'qunit-dom';
+- import { loadTests } from 'ember-qunit/test-loader';
+- import { start, setupEmberOnerrorValidation } from 'ember-qunit';
++ import { start as qunitStart, setupEmberOnerrorValidation } from 'ember-qunit';
+
++ export function start() {
+    setApplication(Application.create(config.APP));
+
+    setup(QUnit.assert);
+    setupEmberOnerrorValidation();
+-   loadTests();
+-   start();
++   qunitStart();
++ }
+```
+
+### Running replacements on... `package.json`
+
+Last but not least, the codemod will modify three sort of things in the `package.json`:
```

Comment:
```suggestion
Last but not least, the codemod will modify three sorts of things in the `package.json`:
```

Reply:
I hate this suggestion 😭 it's one of the "this feels strange so it's probably right" kind of English suggestions 🫠